rest_api/aggregation/database_side/queries.py

```diff
@@ -148,6 +149,37 @@
             'genotype_validation': '$genotype_validation',
             'called_gender': '$called_gender',
             'provided_gender': '$provided_gender',
+            'sample_contamination': '$sample_contamination',
+            'species_contamination': '$species_contamination',
+            'reviewed': '$reviewed',
+            'useable': '$useable',
+            'delivered': '$delivered',
+            'proc_status': '$most_recent_proc.status',
+            'review_comments': '$review_comments',
+            'most_recent_proc': '$most_recent_proc',
+            'run_elements': {'$filter':{
+                'input': '$run_elements',
+                'as': 're',
+                'cond': { '$eq': [ '$$re.useable', 'yes' ] }
```

Comment:
Yes it is.
It seems to be the way  to access sub field in filter
see https://docs.mongodb.com/manual/reference/operator/aggregation/filter/